libs/pmp-web/shared/ui-sidebar/src/lib/components/sidebar/sidebar.component.html

```diff
@@ -5,20 +5,26 @@
         class="sidebar__item"
         [routerLink]="['/reviewer']"
         routerLinkActive="sidebar__item--active"
+        matTooltip="PR reviewers pending"
+        matTooltipPosition="right"
       >
         <mat-icon>people</mat-icon>
       </a>
       <a
         class="sidebar__item"
         [routerLink]="['/repositories']"
         routerLinkActive="sidebar__item--active"
+        matTooltip="Repositories with pending PRs"
```

Comment:
I think that "Repositories" will be enough.
Please replace also Projects with Repositories in the title under the /repositories URL.